### PR TITLE
[ICONS] Volume 9 icons

### PR DESCRIPTION
**Pull request by Shijir**
Tested on: Chrome.

Signed-off-by: Shijir Tsogoo <stsogoo@vmware.com>

### CODE REVIEW

File: src/clarity-icons/shapes/technology-shapes.ts [hunk 1]
```diff
@@ -1363,7 +1363,85 @@ const technologyShapes: any = {
...
+            <path d="M26,5.74A1.74,1.74,0,0,0,24.26,4H3.74A1.74,1.74,0,0,0,2,5.74V20.26A1.74,1.74,0,0,0,3.74,22H4V6H26Z" class="clr-i-solid clr-i-solid-path-1" />
+            <path d="M30,9.74A1.74,1.74,0,0,0,28.26,8H7.74A1.74,1.74,0,0,0,6,9.74V24.26A1.74,1.74,0,0,0,7.74,26H8V10H30Z" class="clr-i-solid clr-i-solid-path-2" />
+            <path d="M19.62,22.6A1.55,1.55,0,1,0,18,21.05,1.6,1.6,0,0,0,19.62,22.6Z" class="clr-i-solid clr-i-solid-path-3" />
+            <path d="M32.26,12H11.74A1.74,1.74,0,0,0,10,13.74V28.26A1.74,1.74,0,0,0,11.74,30H32.26A1.74,1.74,0,0,0,34,28.26V13.74A1.74,1.74,0,0,0,32.26,12ZM19.62,17.74a3.31,3.31,0,1,1-3.38,3.31A3.35,3.35,0,0,1,19.62,17.74Zm0,11.13a7.94,7.94,0,0,1-8-7.82,7.83,7.83,0,0,1,.11-1.29l1.75.3a5.36,5.36,0,0,0-.11,1,6.18,6.18,0,0,0,6.28,6.06,6.35,6.35,0,0,0,5-2.46h2.1A8.06,8.06,0,0,1,19.62,28.87ZM29.89,22.2H24.36V20.44h1.48A6.19,6.19,0,0,0,19.62,15a6.48,6.48,0,0,0-1.41.16l-.45-1.7a8.16,8.16,0,0,1,1.86-.22,8,8,0,0,1,8,7.21h2.26a.88.88,0,0,1,0,1.76Z" class="clr-i-solid clr-i-solid-path-4" />
+        </svg>`,
+
+    "nvme": `
```
Comment (hippee-lee) [inline src/clarity-icons/shapes/technology-shapes.ts]: Just double checking that we want nvme here. I'm not sure what that is.

Comment (Shijir) [reply src/clarity-icons/shapes/technology-shapes.ts]: It's for `non-volatile memory express`. We could probably rename it to `nvm-express` or `nv-memory-express`, maybe?

Comment (Shijir) [reply src/clarity-icons/shapes/technology-shapes.ts]: I see it's is usually abbreviated as "NVMe" or "NVM Express".

Comment (hippee-lee) [reply src/clarity-icons/shapes/technology-shapes.ts]: I think you are good. If you are looking for that you are likely familiar with the acronym.

File: src/clarity-icons/shapes/technology-shapes.ts [hunk 2]
```diff
@@ -1363,7 +1363,85 @@ const technologyShapes: any = {
...
+            <path d="M16.44,31.82a2.15,2.15,0,0,1-.38-2.55l.53-1-1.09-.33A2.14,2.14,0,0,1,14,25.84V23.79a2.16,2.16,0,0,1,1.53-2.07l1.09-.33-.52-1a2.17,2.17,0,0,1,.35-2.52,18.92,18.92,0,0,0-2.32-.16A15.58,15.58,0,0,0,2,23.07v7.75a1,1,0,0,0,1,1H16.44Z" class="clr-i-solid clr-i-solid-path-2" />
+            <path d="M33.7,23.46l-2-.6a6.73,6.73,0,0,0-.58-1.42l1-1.86a.35.35,0,0,0-.07-.43l-1.45-1.46a.38.38,0,0,0-.43-.07l-1.85,1a7.74,7.74,0,0,0-1.43-.6l-.61-2a.38.38,0,0,0-.36-.25H23.84a.38.38,0,0,0-.35.26l-.6,2a6.85,6.85,0,0,0-1.45.61l-1.81-1a.38.38,0,0,0-.44.06l-1.47,1.44a.37.37,0,0,0-.07.44l1,1.82A7.24,7.24,0,0,0,18,22.83l-2,.61a.36.36,0,0,0-.26.35v2.05a.36.36,0,0,0,.26.35l2,.61a7.29,7.29,0,0,0,.6,1.41l-1,1.9a.37.37,0,0,0,.07.44L19.16,32a.38.38,0,0,0,.44.06l1.87-1a7.09,7.09,0,0,0,1.4.57l.6,2.05a.38.38,0,0,0,.36.26h2.05a.38.38,0,0,0,.35-.26l.6-2.05a6.68,6.68,0,0,0,1.38-.57l1.89,1a.38.38,0,0,0,.44-.06L32,30.55a.38.38,0,0,0,.06-.44l-1-1.88a6.92,6.92,0,0,0,.57-1.38l2-.61a.39.39,0,0,0,.27-.35V23.82A.4.4,0,0,0,33.7,23.46Zm-8.83,4.72a3.34,3.34,0,1,1,3.33-3.34A3.34,3.34,0,0,1,24.87,28.18Z" class="clr-i-solid clr-i-solid-path-3" />
+        </svg>`,
+
+    "bitcoin": `
```
Comment (mathisscott) [inline src/clarity-icons/shapes/technology-shapes.ts]: does bitcoin make more sense here or in commerce?

Comment (Shijir) [reply src/clarity-icons/shapes/technology-shapes.ts]: @mathisscott hmm 🤔  the federal reserve hasn't approved Bitcoin as a currency yet. So, until then, it's just another new technology. Jk 😄 . Yeah, I think it should go in Commerce.

**Pull request merged**